src/content/en/2022/fonts.md

```diff
@@ -1,22 +1,851 @@
+
+{{ figure_markup(
+  image="color-font-formats.png",
+  caption="Color font formats.",
+  description="Pie chart showing SVG is the color font used on 74.2% of the time on mobile, COLRv0 is 22.8%, CBDT 2.5%, sbix 0.2% and finally COLRv1 also 0.2%..",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQF6OH_2-0apcFzjE-iHSQNuZqp9DtM7udIeOzPSOSMM-Pf6KdTnRwAclX9QPZF1vNNgu6acZvqoN5b/pubchart?oid=2109618513&format=interactive",
+  sheets_gid="1976816146",
+  sql_file="color_fonts.sql"
+  )
+}}
+
+Taking a look at usage data paints an interesting picture: 79% of color font usage is using SVG, 19% uses COLR v0, and 2% uses CBDT.
+
+We can safely conclude that the image based formats are not popular, and for good reasons: the embedded images don't scale well, and their file sizes are not appropriate for web usage.
+
+The split between the vector color font formats however is more nuanced. While OT-SVG seems to have the upper hand at the moment, COLR still has significant usage. The COLR format has a lot going for it: it is supported by all browsers, it can be used in variable fonts, and it is easy to implement. For those reasons alone, we expect it to become the most popular format. A more cynical take is that it will become the most popular format because Google is <a hreflang="en" href="https://bugs.chromium.org/p/chromium/issues/detail?id=306078">refusing to implement OT-SVG support in Chrome and Android</a>. Interestingly, <a hreflang="en" href="https://lists.webkit.org/pipermail/webkit-dev/2021-March/031765.html">Apple is refusing to implement COLR v1</a>, because a lot of COLR v1 features are already supported by OT-SVG. Unfortunately, web developers are caught in the middle of this "color fonts war". We hope this situation is soon resolved and we can all start using color fonts.
+
+The CSS specification has been updated to support color fonts to allow <a hreflang="en" href="https://css-tricks.com/colrv1-and-css-font-palette-web-typography/">selection and customization of palettes</a>. Palettes are custom color schemes stored in the font by the type designer. The CSS `font-palette` property allows you to select a palette from the font and the `@font-palette-values` rule allows you to create new palettes or override existing ones. One of the more obvious use cases of this technology is to have light and dark mode palettes built right into the color font. There is a lot of unexplored potential there.
+
+{{ figure_markup(
+  image="bradley-initials.png",
+  caption='<a hreflang="en" href="https://tools.djr.com/misc/bradley-initials/">Bradley Initials</a> using COLR v1 and multiple palettes by <a hreflang="en" href="https://djr.com/">David Jonathan Ross</a>.',
+  description="TODO.",
```

Comment:
@bramstein could you add a description (used in alt text) here?

Reply:
Done!